bulgarian/burning_room.md

```diff
@@ -23,3 +23,5 @@
 10. [Решаваш, че си попаднал в Твърде лично и ти си Лиъм Нийсън.](./taken/taken.md)
 
 11. [Протягаш ръка към пистолета, но установяваш че ръцете ти са вързани.](./tied_hands/tied_hands.md)
+
+12. [Решаваш, че ти е писнало от тази примитивна планета. Нега си гори! "Скоти телепортирай ме на Ентърпрайз!"](./Enterprise/Enterprise.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.